Doc-804: Document new skin settings (#1931)
tinymce/tinymce-docs: tinydrive/configuration.md

## Configuring the Tiny Drive UI

### tinydrive_skin

> This options sets the skin applied to {{site.cloudfilemanager}}. The default skin included with {{site.cloudfilemanager}} is named "oxide".

Reviewer: ```suggestion
This option sets the skin applied to {{site.cloudfilemanager}}. The default skin included with {{site.cloudfilemanager}} is named "oxide".
```

> This option allows you to specify the skin that {{site.cloudfilemanager}} should use. The default skin included with {{site.cloudfilemanager}} is named "oxide".

Reviewer 2: ```suggestion
This options sets the skin applied to {{site.cloudfilemanager}}. The default skin included with {{site.cloudfilemanager}} is named "oxide".
```

**Type:** `String`

> **Default Value:** `oxide`
> 
> **Possible Values:** `oxide`, `oxide-dark`

Reviewer: ```suggestion
**Default Value:** `'oxide'`

**Possible Values:** `'oxide'`, `'oxide-dark'`
```

As this is a string, we normally add the quotes so it can be copied/pasted easier.

> **Possible Values:** `oxide`, `oxide-dark`

Reviewer 2: Based on this, I expect that this option doesn't yet allow you to use custom or premium skins?

Author: That's correct, it's not possible to create custom skins for Tiny Drive and the extra skin we provide isn't an add-on.

Reviewer 2: ok. just checking.

> If the `tinydrive_skin` setting is not specified, {{site.cloudfilemanager}} will try and use the skin specified by the [skin]({{ site.baseurl }}/configure/editor-appearance/#skin) setting before falling back to the default "oxide" skin.

Reviewer 2: ```suggestion
If the `tinydrive_skin` option is not specified, {{site.cloudfilemanager}} will try and use the skin specified by the [{{site.productname}} `skin` option]({{ site.baseurl }}/configure/editor-appearance/#skin) before falling back to the default "oxide" skin.
```
